aten/src/ATen/core/jit_type.h

```diff
@@ -787,7 +787,7 @@ struct CAFFE2_API DictType : public Type {
       return getKeyType()->isSubtypeOf(dict_rhs->getKeyType()) &&
           getValueType()->isSubtypeOf(dict_rhs->getValueType());
     }
```

Comment:
We don't need this part now, because `Type::isSubtypeOf` will use `operator==` which contains exactly the same lines.